### PR TITLE
Publish CVE-2020-7331.json

### PR DESCRIPTION
**Pull request by ssarangi1202**
SB is live

### CODE REVIEW

File: 2020/7xxx/CVE-2020-7331.json [hunk 1]
```diff
@@ -1,18 +1,93 @@
 {
```
Comment (cve-team) [inline 2020/7xxx/CVE-2020-7331.json:1]: 'vendor_name' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by cristina479** [comment]
@ssarangi1202 Please resolve the conflicts and address the validation errors so that we can publish the changes of CVE-2020-7331.

**Comment by ssarangi1202** [comment]
I have made the changes and have updated the CVE. Kindly approve.

File: 2020/7xxx/CVE-2020-7331.json [hunk 2]
```diff
@@ -13,28 +13,33 @@
...
+                                "product_name": "McAfee Endpoint Security (ENS) ",
                                 "version": {
                                     "version_data": [
                                         {
                                             "version_affected": "<=",
                                             "version_name": "10.7.x",
                                             "version_value": "10.7.0 September 2020 Update"
+                                        },
+                                        {
+                                            "version_affected": "<=",
```
Comment (cristina479) [inline 2020/7xxx/CVE-2020-7331.json]: If version <= 10.6.1 September 2020 Update are affected then they should also be included in the description

**Pull request merged**